index.css

```diff
@@ -0,0 +1,88 @@
+body {
+    font-family: Arial, sans-serif;
+    background: #f5f5f5;
+}
+
+#todo-app {
```

Comment:
Don't use IDs in CSS, use classes instead so we can compose multiple classes together. -1